### ISSUE TITLE
Wrong results from running "openssl speed -evp aes-256-cbc" in Emscripten

### ISSUE THREAD

**Issue by DevelopDaily** [openssl/openssl#18216]
When it is run in the native system (Ubuntu 20.40), the first line of the result looks like this:

```
openssl speed -evp aes-256-cbc 
Doing aes-256-cbc for 3s on 16 size blocks: 26616414 aes-256-cbc's in 1.46s
```

But when it is run in my Web Assembly build, there are two problems: 1) it is very slow (waiting for minutes), 2) the result seems to be wrong:

```
node openssl speed -evp aes-256-cbc
Doing AES-256-CBC for 3s on 16 size blocks: 2147483647 AES-256-CBC's in 0.00s
```

What does the "0.00s" mean?

Here is how I built the OpenSSL (the latest from the GitHub) in Web Assembly:

```
emconfigure ./Configure linux-generic64 -no-asm -no-threads -static -no-afalgeng -DOPENSSL_SYS_NETWARE -DSIG_DFL=0 -DSIG_IGN=0 -DHAVE_FORK=0 -DNO_SYSLOG
sed -i 's|^CROSS_COMPILE.*$|CROSS_COMPILE=|g' Makefile
emmake make
```

**Comment by t8m** [comment]
Most probably there is some problem with timers in your environment where the command is run. I do not think there is a bug in the command itself.

**Comment by DevelopDaily** [comment]
@t8m 
Could you please tell me how to run an OS command or something to verify if the timers are running correctly?

**Comment by t8m** [comment]
Unfortunately I do not know of any easy way how to verify that apart from writing some test programs.

**Comment by nhorman** [comment]
Marking as inactive, to be closed when 3.4 dev cycle ends, barring further input
